src/utils/interfaces/object-abstract-class.ts

```diff
@@ -1,5 +1,6 @@
 
 export abstract class BaseObject {
+  [x: string]: any;
```

Comment:
Any is generally bad practice in typescript. You are saying base object can now have any property of any type. Essentially disabling typescript.

```suggestion
```

Do you remmeber which field was erroring? Was it the pipe listener?
its better to just add that using the arrow syntax